WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingView.swift

```diff
@@ -183,7 +187,7 @@ private extension StoreOnboardingView {
 private extension StoreOnboardingView {
     enum Layout {
         static let insetsWithViewAllButton: EdgeInsets = .init(top: 16, leading: 16, bottom: 16, trailing: 16)
-        static let insetsWithoutViewAllButton: EdgeInsets = .init(top: 16, leading: 16, bottom: 0, trailing: 16)
+        static let insetsWithoutViewAllButton: EdgeInsets = .init(top: 16, leading: 16, bottom: 1, trailing: 16)
```

Comment:
The 0 padding causes an issue with the layout of the view when embedded in the dashboard view. Setting a non-zero layout is a workaround to fix it.